Update Readme to be more readable (#134)
faros-ai/airbyte-local-cli: README.md

---

### 🚀 Step 1. Install

[All releases](https://github.com/faros-ai/airbyte-local-cli/releases)

#### MacOS (Arch: Apple Silicon)

Here is the steps of downloading the CLI on MacOS. Linux should have very similar steps.

```sh
wget -O airbyte-local.zip https://github.com/faros-ai/airbyte-local-cli/releases/download/v0.0.5/airbyte-local-macos-arm64.zip
unzip -o airbyte-local.zip

./airbyte-local
```

### Linux
```sh
wget -O airbyte-local.zip https://github.com/faros-ai/airbyte-local-cli/releases/download/v0.0.5/airbyte-local-linux-x64.zip
unzip -o airbyte-local.zip

./airbyte-local
```

#### Windows (Powershell)

```ps1
Invoke-WebRequest -Uri "https://github.com/faros-ai/airbyte-local-cli/releases/download/v0.0.5/airbyte-local-win-x64.zip" -OutFile "airbyte-local-win-x64.zip"
Expand-Archive -Path "airbyte-local-win-x64.zip" -DestinationPath . -Force

.\airbyte-local
```

### 🧩 Step 2. Create an Airbyte Config File

Before you can run your sync, you need to tell the CLI:

* **Where to pull data from** (e.g., GitHub, Jira)
* **Where to push data to** (usually Faros)

You have two options to create this configuration:

#### ✨ Option A: Auto-Generate (Recommended for new users)

> Use this if you’re new or want a quick start.

Reviewer: ```suggestion
Use this if you’re new or want to get started quickly.
```

Run:
```bash
./airbyte-local generate-config <source> [destination]
```

Examples:
```bash
# Pull from GitHub → Push to Faros
./airbyte-local generate-config github

# Pull from Jira → Push to Faros
./airbyte-local generate-config jira
./airbyte-local generate-config -s jira

> # Use your own custom or non Faros managed images

Reviewer: ```suggestion
# Use your own custom or non Faros-managed images
```

./airbyte-local generate-config --image farosai/airbyte-github-custom-source
./airbyte-local generate-config --image farosai/airbyte-github-custom-source farosai/airbyte-custom-destination

# For help
./airbyte-local generate-config --help
```

This command will:

- Print source/destination config tables in your terminal.
- Create a `faros_airbyte_cli_config.json` file with **required** fields only.
- Show you 🔹 Next Steps in the terminal — follow those to continue!

> Pro tips:

Reviewer: ```suggestion
Tips & Tricks:
```

- Source and destination names are case-insensitive and tolerate typos.
- Destination defaults to Faros unless specified
- Add `-s` to skip printing the config tables if you just want the file.

> - Use `--image` to specify your custom images. This doesn't tolerate typos and set default destination to Faros, i.e. `farosai/airbyte-faros-destination`.

Reviewer: ```suggestion
- Use `--image` to specify your custom images. This doesn't tolerate typos and doesn't set the default destination to Faros, i.e. `farosai/airbyte-faros-destination`.
```

Author: it does set the default to faros destination image

Reviewer: Ack. Updated my suggestion.
